README.md

```diff
@@ -94,14 +94,18 @@ Around 50% of the App is written in [Kotlin](https://kotlinlang.org/). Kotlin is
 Generally, projects are created using package by layer approach where packages are names by layers like `ui`, `activity`, `fragment`, etc but it quickly becomes unscalable in large projects where a large number of unrelated classes are crammed in one layer and it becomes difficult to navigate through them.  
 Instead, we follow package by feature, which at the cost of flatness of our project, provides us packages of isolated functioning related classes which are likely to be a complete self-sufficient component of the application. Each package all related classes of view, presenter, their implementations like Activities and Fragments.  
 A notable exception to this is the `helper` module and data classes like Models and Repositories as they are used in a cross component way.  
-***Note:** The interface contract for Presenter and View is present in `contract` package in each module`*
```

Comment:
Do not remove asterisk (*) from end. It is used to give overall italic effect. i.e 
```
italic ( bold ( "Note:" ) + "The interface contract for Presenter and View is present in `contract` package in each module" )
```